.ci/packaging.groovy

```diff
@@ -397,31 +340,3 @@ def withBeatsEnv(Closure body) {
     }
   }
 }
-
-/**
-* This method fixes the filesystem permissions after the build has happenend. The reason is to
-* ensure any non-ephemeral workers don't have any leftovers that could cause some environmental
-* issues.
-*/
-def deleteWorkspace() {
-  catchError(buildResult: 'SUCCESS', stageResult: 'SUCCESS') {
-    fixPermissions()
-    deleteDir()
-  }
-}
-
-def fixPermissions() {
```

Comment:
this is because the test run as root in Docker containers and some files has wrong permissions on the host filesystem, Is it resolved? if not, it is still needed

Reply:
No more static workers but ephemeral workers, so `arm` used to be static but not anymore. macos supports has not been used much, in addition, the only static worker we had at the time with the configuration was gone :/

So I'd say we don't need this anymore as we use ephemeral workers

Reply:
this is not only to keep the workspace sane, but it is also to be able to access some files in the workspace after run the tests, IIRC the JUnit files or some test results

Reply:
The build didn't fail, so I assume since it used ephemeral workers, then we don't need use it.

In fact, it was added to support `arm` while `arm` were static based:
- https://github.com/elastic/beats/pull/23592